python/Ganga/new_tests/GPI/Tasks/TestCoreTasks.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+from ..GangaUnitTest import GangaUnitTest
+
+
+class TestCoreTasks(GangaUnitTest):
+    """A set of tests to cover both ITask and CoreTask plus related objects"""
+
+    def setUp(self):
+        """Make sure that the Tasks object isn't destroyed between tests"""
+        super(TestCoreTasks, self).setUp()
+        from Ganga.Utility.Config import setConfigOption
+        setConfigOption('TestingFramework', 'AutoCleanup', 'False')
+
+    def test_a_TaskCreation(self):
+        """ First construct a Task object with Transform, etc."""
+        from Ganga.Utility.Config import getConfig
+        self.assertFalse(getConfig('TestingFramework')['AutoCleanup'])
+
+        from Ganga.GPIDev.Base.Proxy import isType
+        from Ganga.GPI import tasks, CoreTask, CoreTransform, Executable, GenericSplitter
+        t = CoreTask()
+
+        trf = CoreTransform()
+        trf.application = Executable()
+        trf.unit_splitter = GenericSplitter()
+        trf.unit_splitter.attribute = "application.args"
+        trf.unit_splitter.values = [ 'arg 1', 'arg 2', 'arg 3' ]
```

Comment:
PEP8 formatting. [There shouldn't be spaces inside brackets](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements). On a few other lines in the tests too.